CMakeLists.txt

```diff
@@ -921,3 +922,4 @@ else()
         message(STATUS "Cannot find the sphinx-build executable.")
     endif()
 endif()
+
```

Comment:
Please restore the unncessary changes in the CMakeLists.txt.

Reply:
restored